### PR TITLE
use repeat actions for beta

### PR DESCRIPTION
**Pull request by benjamin-confino**
Ensure we test with and without beta. Follow up to #26278

### CODE REVIEW

**Comment by benjamin-confino** [comment]
#build

**Comment by benjamin-confino** [comment]
#build

**Comment by benjamin-confino** [comment]
#build

**Comment by benjamin-confino** [comment]
#build

**Comment by benjamin-confino** [comment]
#build

File: dev/io.openliberty.microprofile.telemetry.1.0.internal_fat/publish/servers/Telemetry10Spi/jvm.options.orig [hunk 1]
```diff
@@ -0,0 +1 @@
+-Dio.opentelemetry.context.enableStrictContext=true
```
Comment (Azquelt) [inline dev/io.openliberty.microprofile.telemetry.1.0.internal_fat/publish/servers/Telemetry10Spi/jvm.options.orig]: This shouldn't be here.

**Review by Azquelt — APPROVED** [review]
Looks good, thank you

**Comment by benjamin-confino** [comment]
#build

**Comment by benjamin-confino** [comment]
Green build: 
https://wasrtc.hursley.ibm.com:9443/jazz/resource/itemOid/com.ibm.team.build.BuildResult/_Rn8ssFbQEe63MKOTK2E49A

**Comment by benjamin-confino** [comment]
#libby

**Pull request merged**